### ISSUE TITLE
Discord

### ISSUE THREAD

**Issue by FeSade1** [valolrant/valolrant#1]
**Can u please add me on discord? I realy want your help. Sade#5403**

**Comment by infective-dev** [comment]
no fuck off kid. learn to use it fgt

**Comment by ghost** [comment]
^^

**Comment by an0nym3sh** [comment]
any clue how to compile?

**Comment by andbeyond711** [comment]
> any clue how to compile?

Yes, all you need is visual studios, and you build as release x86. Make sure everything is set up with interception driver though. It will not work if you haven't followed all steps.

**Comment by an0nym3sh** [comment]
which file in particular we have to compile tho? also, will the compilation result into a dll file that i have further inject?

**Comment by andbeyond711** [comment]
> which file in particular we have to compile tho? also, will the compilation result into a dll file that i have further inject?
The only file you need to open and compile is the dll2.sln. Once opened in visual studios, build the project as x86 release. You will then get a dll file in your folder that you put into the directory of the program you are injecting into(NOT VALORANT)(BE CREATIVE AND USE ANY 32 BIT PROGRAM!). Im not listing all of the instructions but those are some.

**Comment by Scrut1ny** [comment]
I know how to do all this shit but what VSC installer extensions do I need for this to compile properly?

**Comment by ixyz400** [comment]
> I know how to do all this shit but what VSC installer extensions do I need for this to compile properly?

you see the code language?
